src/time_unix.c

```diff
@@ -40,13 +40,13 @@ extern "C"
 #include "rcutils/allocator.h"
 #include "rcutils/error_handling.h"
 
-#if !defined(__MACH__)  // Assume clock_get_time is available on OS X.
+#if !defined(__MACH__)  && !defined(__APPLE__)   // Assume clock_get_time is available on OS X.
```

Comment:
The linter is unhappy here.

```suggestion
#if !defined(__MACH__) && !defined(__APPLE__)   // Assume clock_get_time is available on OS X.
```